README.md

```diff
@@ -21,6 +21,19 @@ automatically refer to the new version when calling the command line scripts.
 pip install -e .
```

Comment:
The ``pip install -e`` is redundant: the ``pip3`` version of the command introduced below should work on both machines with python 2 and 3 as well as those that only have python 3.

Reply:
It does not. With both installed, i get the following error when running `pip install -e .`:

```
Obtaining file:///media/data/git/obs/scripts
Collecting aabbtree==2.6.2 (from obs==0.0.1)
  Using cached https://files.pythonhosted.org/packages/a9/53/19c744fca03901c996cd8b38408b86725f9738e9daa4d12a53d9f5ddb491/aabbtree-2.6.2-py2.py3-none-any.whl
Collecting coloredlogs==15.0 (from obs==0.0.1)
  Using cached https://files.pythonhosted.org/packages/84/a6/837dbf6eac344cb74f0ba86b79e8180855570af3e26bcb1ea5f650cf944c/coloredlogs-15.0-py2.py3-none-any.whl
Collecting geopy==2.2.0 (from obs==0.0.1)
  Could not find a version that satisfies the requirement geopy==2.2.0 (from obs==0.0.1) (from versions: 0.93, 0.94, 0.94.1, 0.94.2, 0.95, 0.95.1, 0.96.0, 0.96.1, 0.96.2, 0.96.3, 0.97, 0.97.1, 0.98, 0.98.1, 0.98.2, 0.98.3, 0.99, 1.0.0, 1.0.1, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.2.0, 1.3.0, 1.4.0, 1.5.0, 1.6.0, 1.6.1, 1.7.0, 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.11.0, 1.12.0, 1.13.0, 1.14.0, 1.15.0, 1.16.0, 1.17.0, 1.18.0, 1.18.1, 1.19.0, 1.20.0, 1.21.0, 1.22.0, 1.23.0)
No matching distribution found for geopy==2.2.0 (from obs==0.0.1)

```

Reply:
Yes, but with only py3 installed, pip3 install still works. It also works with both py2 and py3 installed. So pip3 should always work and we could ditch the **non-3 line** as well as the "If this then" line. I agree that pip (without 3) will fail on a dual setup. The line you introduced should work on all setups.